pkg/machines/components/storagePools/storagePool.jsx

```diff
@@ -163,6 +170,10 @@ class StoragePoolActions extends React.Component {
 
         return (
             <React.Fragment>
+                { storagePool.active &&
+                <Button id={`deactivate-${id}`} onClick={this.onDeactivate.bind(this)}>
+                    {_("Deactivate")}
```

Comment:
Same here as for the `Activate` name, for consistency I would prefer `Stop` as opposite to `Start`.

Reply:
Same as above, I am ok to change it for consistency. But since we can change the other sentence as well, let's pick the better naming now that it's not really established.

@garrett what do you think?
Storage Pools start/stop vs activate/deactivate
I guess it's a subtle difference anyway.